Game_Doc.md

```diff
@@ -0,0 +1,50 @@
+# Concentration Game
+## Description
+A simple concentration card game where the user flips cards to reveal it's icon and must find that cards duplicate by flipping another card.
+
+## Components
+* [Board Code](./components/board.tsx) / [Doc](#board)
+* [Card Code](./components/card.tsx) / [Doc](#card)
+
+## Board
+### Usage
+```ts
+<Board size={[4, 3]} />
+```
+
+The width property must be even, equal/more than 2 and no more than 6. <br>
+The height propery must be equal or more than 1 and less than or equal to 6.
+<br><br>
+The Board component uses the Boardprops interface referenced below.
+
+```ts
+  interface BoardProps {
+    size: [width: number, height: number]
+  }
+```
```

Comment:
```suggestion
### Usage

The Board component uses the `BoardProps` interface referenced below.

interface BoardProps {
  size: [width: number, height: number]
}

<Board size={[4, 3]} />

* The `width` property must be an **even** integer between 2 and 6.
* The `height` property must be an integer between 2 and 6.
```

Made it a little less wordy and revered the order so the Props interface was introduced BEFORE referring to the array positions as "width" and "height".

_Also, please add the syntax highlighting back to the code snippets - I had to remove them because the Suggestions syntax breaks if you include a triple tick (```) in the suggestion_ 😬